### PR TITLE
Align Allocated Codecache by 64-Bytes on Power 

### PR DESCRIPTION
**Pull request by AlenBadel**
16-byte prefix instructions were introduced
with P10 support.
These prefix instructions need to be placed inside
the same 64-byte block in memory,
otherwise, they will be interpreted as two separate instructions.
This PR 64-byte aligns the allocated code cache memory so that
each AOT and JIT load and compilation respectively can start
at the same boundary.

Signed-off-by: AlenBadel <Alen.Badel@ibm.com>

### CODE REVIEW

File: runtime/compiler/runtime/codertinit.cpp
Comment (fjeremic) [inline runtime/compiler/runtime/codertinit.cpp:223]: I suggest we add a comment, similar to Z, on why this value was chosen. Sure someone can use `git blame` and trace back the history. But as files move around, get renamed, refactored, etc. these things can be hard to track. It is more informative to just put a short comment right in the code so future readers will immediately understand why the alignment is needed, and specifically a 64-byte alignment.

**Comment by fjeremic** [comment]
See CONTRIBUTING.md on how to consistently format commit messages. This makes reading `git log` much more pleasant. See [linked resources](https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md#other-resources-for-writing-good-commits) on why this is important.

>The first line should be less than 70 characters, where reasonable, and should be written in sentence case (capitalize the first letter) preferably not ending in a period. Leave a blank line between the first line and the message body.
>
>**The body should be wrapped at 72 characters, where reasonable.**

https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md#commit-guidelines

File: runtime/compiler/runtime/codertinit.cpp
Comment (AlenBadel) [reply runtime/compiler/runtime/codertinit.cpp]: Done. Would you like to see comments added on the X86 32-byte alignment? To my knowledge, this was to align to a 32-byte cache line on earlier processors.

Comment (fjeremic) [reply runtime/compiler/runtime/codertinit.cpp]: Sure! If you've dug up on the history and that is the case I think that would be very helpful to future readers. Thanks for making the change!

**Review by fjeremic — APPROVED** [review]
Looks good to me. Bonus points if we can add comments for x86, but that shouldn't block this PR. Will let someone with Power expertise handle committer merging process.

**Pull request merged**
